### PR TITLE
Development_Setup.rst: Update "Installing from Git"

### PR DESCRIPTION
**Pull request by EverWinter23**
Closes https://github.com/coala/coala/issues/3903

### CODE REVIEW

**Comment by EverWinter23** [comment]
Commit message was going to be:
Development_Setup.rst: Update "Installing from Git"

But it was too long.

File: docs/Developers/Development_Setup.rst [hunk 1]
```diff
@@ -91,30 +91,25 @@ https://gitlab.com/coala/coala-utils/
 Installing from Git
 -------------------
 
-We recommend first installing the latest development snapshot of coala's
-master branch from and all of its dependencies with pip3 using
+We recommend installing coala from the source and its dependencies using
+pip3.
```
Comment (manankalra) [inline docs/Developers/Development_Setup.rst]: change it to _**with pip3 using**_

File: docs/Developers/Development_Setup.rst [hunk 2]
```diff
@@ -91,30 +91,25 @@ https://gitlab.com/coala/coala-utils/
...
-    (coala-venv)$ pip3 install -e <path/to/forked/repository>
-
-You will then be able to edit the repository and have the changes take effect
-in your virtualenv immediately. You will also be able to use pip3 to manage
-your installation of the package should you need to install from a different
-source in the future.
+Once you have forked the repository, you will be able to edit the
+repository. This will also help you in managing the installed package
+using pip3 should you need to install from a different source in the
+future.
```
Comment (manankalra) [inline docs/Developers/Development_Setup.rst]: For this part, I guess it is required to just remove the second code sample and add an additional explanation for the first code sample. The text section after the second code sample should be kept imo.

Comment (EverWinter23) [reply docs/Developers/Development_Setup.rst]: Ok

Comment (manankalra) [reply docs/Developers/Development_Setup.rst]: > add an additional explanation for the first code sample
The issue thread clearly specifies everything.

**Comment by manankalra** [comment]
> add an additional explanation for the first code sample.

The issue thread clearly specifies everything.
Also, you need to squash your commits into one.

**Comment by EverWinter23** [comment]
I have already squashed my commits into one. Pls check again.

**Comment by manankalra** [comment]
@EverWinter23 They aren't squashed.

**Comment by EverWinter23** [comment]
Sorry, didn't see those. Making changes ASAP.

**Pull request closed without merging**